pyproject.toml

```diff
@@ -115,6 +115,14 @@ sphinx = "6.1.3"
 tensorflow = {version = "2.12.0", markers="sys_platform != 'darwin'"}
 tensorflow-macos = {version = "2.12.0", markers="sys_platform == 'darwin'"}
 tensorflow-metal = {version = "0.8.0", markers="sys_platform == 'darwin' and platform_machine == 'arm'"}
+tensorflow-io-gcs-filesystem = [
+  {version = "0.34.0"},
+]
+# tensorflow-io-gcs-filesystem = [
+#   {version = "0.35.0", markers="sys_platform == 'darwin' and platform_machine != 'arm' and python_version >= '3.9'"},
+#   {version = "0.35.0", markers="sys_platform != 'darwin' and python_version >= '3.9'"},
+#   {version = "0.35.0", markers="sys_platform != 'darwin' and python_version == '3.8' and platform_machine != 'arm'"}
+# ]
```

Comment:
We could also have something like that but for some reason it didn't seem to work.